pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: convert
-version: 3.1.2-dev
+version: 4.0.0-dev
```

Comment:
(This is going to be hard as long as `test` depends on someting which depends on `^3.0.0`.
Might be better to just declare this change "non-breaking".)

Reply:
We could discuss at the ecosystem meeting tomorrow? Clearly changing a data mapping is breaking but it is hard to imagine people being negatively impacted. For now why don't you change it back so that you can write tests?

Reply:
I think that we decided to use a 3.X version.